frontend/src/scenes/exports/ExportsList.tsx

```diff
@@ -124,6 +135,18 @@ export function Exports(): JSX.Element {
                                         icon={export_.paused ? <IconPlay /> : <IconPause />}
                                         tooltip={export_.paused ? 'Resume this BatchExport' : 'Pause this BatchExport'}
                                         loading={pausing || resuming}
+                                        disabled={deleting}
+                                    />
+                                    <LemonButton
+                                        status="primary"
+                                        type="secondary"
+                                        onClick={() => {
+                                            deleteExport()
+                                        }}
+                                        icon={<IconDelete />}
+                                        tooltip="Permanently delete this BatchExport"
```

Comment:
fwiw I implemented this as a soft delete, although I don't think the user should care about that. Nothing to change.

Reply:
oh except maybe a space between Batch and Export!

Reply:
Although, temporal doesn't have a concept of soft deletes (unless we are pausing the schedule), so an undo may not be possible 🤔

Reply:
I guess you can pretty much get the functionality by using pause anyway. I think no need to do anything more that what you've done, but just wanted to make sure you knew the implementation in case it's relevant.